specification/network/resource-manager/Microsoft.Network/stable/2020-06-01/virtualWan.json

```diff
@@ -5305,6 +5440,23 @@
         }
       }
     },
+    "VpnGatewayPacketCaptureStartParameters": {
+      "properties": {
+        "filterData": {
+          "type": "string",
```

Comment:
Based on the example, should it be an object and include right schema?

Reply:
Hi Qiu, there is an object inside this string, the manager service inside resource provider will handle this string itself

Reply:
This is modeled as a JSON to have more flexibility in publishing changes to filtering data. We are going to publish this JSON schema in the public domain for the customer to refer to. We also plan to provide helper PowerShell command to generate JSON

Reply:
Hi Qiu, I just talked to the Author of the service, he said " its a JSON string and there is a precedence for it and also the JSON schema example are present in the public domain. there is also an example for Start-VirtualNetworkGatewayPacketCapture https://docs.microsoft.com/en-us/azure/vpn-gateway/packet-capture"

Reply:
What's the meaning of public domain? Is it a public standard?

Reply:
Hi Fanqiu, it is not a standard but a publicly available information, here is the information  https://docs.microsoft.com/en-us/powershell/module/az.network/start-azvirtualnetworkgatewaypacketcapture?view=azps-4.2.0

Reply:
@raych1 @lirenhe Hi. Could you share your opinion on this property? Is it ok to merge this one?

Reply:
@jiaczh , can you share the reason why not define the model for the JSON? Is it a fixed schema?

Reply:
@raych1 , Hi Ray, this is publicly available information in the public domain, many websites have link to the scheme of this model.

Reply:
Here is the PR link that discuss the same issue, https://github.com/Azure/azure-rest-api-specs/pull/7151

Reply:
Suggest to add CLI support to generate JSON.

Reply:
@raych1  Hi Ray, I ask the backend author and yes he does plan to add CLI support in the near future

Reply:
@jiaczh Do you have any official doc to explain the schema of this filter data other than PS's doc? Better to put the link into decription.

Reply:
Hi @MyronFanQiu , We are currently working  with PM to get the official documentation, but it is not publicly available yet, I sent you a one-note link  through teams, you can take a look

Reply:
better to refine description after the release of official documentation.